modules/issue/template/unmarshal.go

```diff
@@ -128,9 +128,18 @@ func unmarshal(filename string, content []byte) (*api.IssueTemplate, error) {
 			}
 		}
 		for i, v := range it.Fields {
+			// set default id value
 			if v.ID == "" {
 				v.ID = strconv.Itoa(i)
 			}
+			// set default submit value
+			if v.Visible == nil {
+				v.Visible = []api.IssueFormFieldVisible{api.IssueFormFieldVisibleForm}
+				// markdown is not submitted by default
+				if v.Type != api.IssueFormFieldTypeMarkdown {
+					v.Visible = append(v.Visible, api.IssueFormFieldVisibleContent)
+				}
```

Comment:
Ouch.
Sounds like it will lead to bugs rather sooner than later

Reply:
I agree, but I think this is here just for backwards compatibility.

Reply:
Oh, I completely misread that line the first time.
I read it as `if visible.Present { visible.append(api.IssueFormFieldVisibleContent) }` which is something completely different.
But yes, my point still kinda stands surprisingly :grin:

Reply:
I dislike it also ... but as long as we want to stay compatible with github we have to do it ...

... so that's why i did add the code comments all over the code handling this case